docs/guides/environments.md

```diff
@@ -5,12 +5,14 @@ these can be specified in the floyd [run](../commands/run.md) command using the
 `--env` option.
 
 If no `--env` is provided, it uses the `keras` image by default, which comes with Python
-3, Keras 2.0.4 and Tensorflow 1.1.0 pre-installed.
+3.6, Keras 2.1.3 and Tensorflow 1.5.0 pre-installed.
 
 | Framework | Env name (--env parameter)  |  Description              | Docker Image |
 | --------- | ------------------ | ------------------------ |-------------|
-| Keras | keras      | Tensorflow 1.1.0 + keras 2.0.6 on Python3.5. |  |
-|       | keras:py2  | Tensorflow 1.1.0 + keras 2.0.6 on Python2. |  |
+| Keras | keras      | Tensorflow 1.5.0 + keras 2.1.3 on Python3.6. |  |
```

Comment:
Do we need to support a separate environment called Keras? Can this not fold under Tensorflow?

Reply:
@houqp

Reply:
yeah, we can fold it now. it wasn't the case when we created it.

Reply:
Sure!